docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.4'
+services:
+  qmk:
+    build:
+      context: .
+      args:
+        FORK: ${FORK}

```

Comment:
Do we really need this compose file yet? It seems to make arg passing more explicit, I like it.

Reply:
Compose is way cleaner than running plain docker commands with all the parameters.
So instead of `docker compose build` in action.yml you would end up with an ugly multiline docker build command with all the compose settings as arguments